src/main/java/com/linkedin/kmf/partitioner/KMPartitioner.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.partitioner;
+
+public interface KMPartitioner {
+
+  int getPartitionForKey(String key, int numPartitions);
```

Comment:
Can we name this method `partition(...)` since the open source Partitioner use this method name?